docs/release/release-team.md

```diff
@@ -67,11 +67,17 @@ As noted above, making changes to  the CAPI release cadence is out of scope for
 
 ## Team Roles
 
-- **Release Lead**: responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring that a retrospective happens. The lead is also responsible for ensuring a successor is selected and trained for future release cycles.
-- **Communications/Docs/Release Notes Manager**: Responsible for communicating key dates to the community, improving release process documentation, and polishing release notes. Also responsible for ensuring the user-facing Netlify book and provider upgrade documentation are up to date.
-- **CI Signal/Bug Triage/Automation Manager**: Assumes the responsibility of the quality gate for the release and makes sure blocking issues and bugs are triaged and dealt with in a timely fashion. Helps improve release automation and tools.
-- **Team member**: Any Release Team lead or manager may select one or more additional members to help with their tasks. These team members will help fulfill future Release Team staffing requirements and continue to grow the CAPI community in general.
-*Note*: This is also documented in [Release tasks](./release-tasks.md) together with a mapping to specific tasks.
+**Notes**:
+
+* The examples in these documents are based on the v1.6 release cycle.
+* The documents focus on tasks that are done for every release. One-time improvement tasks are out of scope.
```

Comment:
Does this still make sense as is?

Reply:
I removed this line as it is not relevant anymore.